client/src/views/mapping/components/inputColumn/index.tsx

```diff
@@ -0,0 +1,63 @@
+import { Button, Card, Elevation, Tag } from "@blueprintjs/core";
+import * as React from "react";
+import { Subscription } from "react-apollo";
+
+import { ISelectedSource } from "../../../../types";
```

Comment:
can you use absolute imports instead ?

Reply:
Ok, je rajoute ce qu'il faut dans le tsconfig pour le faire